gym/lib/gym/options.rb

```diff
@@ -230,8 +231,18 @@ def self.plain_options
                                      description: "Suppress the output of xcodebuild to stdout. Output is still saved in buildlog_path",
                                      optional: true,
                                      type: Boolean),
+
+        FastlaneCore::ConfigItem.new(key: :xcodebuild_formatter,
+                                     env_names: ["GYM_XCODEBUILD_FORMATTER", "FASTLANE_XCODEBUILD_FORMATTER"],
+                                     description: "xcodebuild formatter to use (ex: 'xcbeautify', 'xcbeautify --quieter', 'xcpretty', 'xcpretty -test')",
```

Comment:
Can we also use no formatter? `disable_xcpretty` is now deprecated in favour of `xcodebuild_formatter`, but it does not seem to come with a disabled option?

Reply:
@AvdLee Oops! That description is supposed to mention just set to an empty string to disable output 🤦‍♂️ I'll update that and make sure that works for `gym` and the other tools!